web/src/components/SearchFeedback.tsx

```diff
@@ -35,6 +35,7 @@ const SearchFeedback = ({ cityCode, languageCode, query, noResults }: SearchFeed
           routeType={SEARCH_ROUTE}
           query={query}
           noResults={noResults}
+          isPositive={null}
```

Comment:
:question: should be "false", shouldn't it?

Reply:
You're right, I thought it should be null for the search but looking at the existing functionality, we send a negative feedback

Reply:
After some research I found that the existing functionality I looked at is from a year ago and that we changed the search feedback to be neutral a while ago. I think it makes more sense for it to be negative though, so I'll open an issue for that.